app/views/shared/_movements.html.haml

```diff
@@ -38,6 +38,9 @@
                 - op = { method: :put, data: { confirm: t(".delete_reason") } }
                 = link_to(delete_reason_transfer_path(mv.transfer), op) do
                   %span.glyphicon.glyphicon-trash
-            %td= seconds_to_hm(mv.amount)
+            %td
+              - positive = mv.amount.positive?
+              %strong{ :class => (positive ? "text-success" : "text-danger") }
```

Comment:
`positive?` is available in ruby `2.3.0` but not in `2.2.2` which is the version that we use in travis.

https://github.com/coopdevs/timeoverflow/pull/206 fix this

grande @sseerrggii :clap: